backend/app/api/endpoints/adapter/tasks.py

```
@@ -684,3 +684,140 @@ def delete_task_services(
     db.refresh(task)
 
     return {"app": app_data}
+
+
+@router.get("/{task_id}/workspace/files", response_model=dict)
+async def get_workspace_files(
+    task_id: int,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)